File: problem_forty_six.js
// 2 array item console sortly in one list
var lst =[3,5,6,3,3,64,64,34,64,88]
var lst1 = [8,7,9,1,2]
function soRt(arr,arr1){
    var array = arr.concat(arr1)
    for (var i =0;i<array.length;i++){
        for(var j= i; j<array.length; j++){
            if(array[i]> array[j]){
                T = array[i]
                array[i]=array[j]
                array[j]= T
            }
        }
    }
    return array  
}

print(Sort_list(lst1,lst2))

File: problem_forty_six.py
# two list item join and print sortly 
lst1 =[1458,5,4,86,86,56]
lst2=[6,8,97,56,4,3]
def Sort_list (lst1,lst2):
    value = lst1+lst2
    for i in range(len(value)):
        for j in range(i,len(value),1):
            if value[i]> value[j]:
                T = value[i]
                value[i]=value[j]
                value[j]=T

    return value            

print(Sort_list(lst1,lst2))